### PR TITLE
Follow up to: #2191 Switch ColorPicker to use tracker_ref 

### PR DESCRIPTION
**Pull request by kmahone**
#2191 got merged before all PR feedback was addressed. 

This change updates the use of local variables assigned to objects from tracker_ref to use 'auto&&' instead of 'auto'. This allows the local use of the tracked object without an addref/release call.

### CODE REVIEW

File: dev/ColorPicker/ColorSpectrum.cpp [hunk 1]
```diff
@@ -574,7 +574,8 @@ void ColorSpectrum::UpdateColorFromPoint(const winrt::PointerPoint& point)
 
 void ColorSpectrum::UpdateEllipse()
 {
-    auto selectionEllipsePanel = m_selectionEllipsePanel.get();
+    auto&& selectionEllipsePanel = m_selectionEllipsePanel.get();
+
```
Comment (StephenLPeters) [inline dev/ColorPicker/ColorSpectrum.cpp:578]: nit: a sneaky tab!

Comment (kmahone) [reply dev/ColorPicker/ColorSpectrum.cpp:578]: eagle eyes. :)

**Pull request merged**